cpp/arcticdb/version/python_bindings.cpp

```diff
@@ -136,6 +136,24 @@ void register_bindings(py::module &version, py::exception<arcticdb::ArcticExcept
         .def_property_readonly("symbol", &VersionedItem::symbol)
         .def_property_readonly("version", &VersionedItem::version);
 
+    py::class_<DescriptorItem>(version, "DescriptorItem")
+        .def_property_readonly("symbol", &DescriptorItem::symbol)
+        .def_property_readonly("version", &DescriptorItem::version)
+        .def_property_readonly("start_index", &DescriptorItem::start_index)
+        .def_property_readonly("end_index", &DescriptorItem::end_index)
+        .def_property_readonly("creation_ts", &DescriptorItem::creation_ts)
+        .def_property_readonly("timeseries_descriptor", [](DescriptorItem& self) {
+          py::object pyobj;
+          if (self.timeseries_descriptor().has_value()) {
+               arcticdb::proto::descriptors::TimeSeriesDescriptor tsd;
+               self.timeseries_descriptor()->UnpackTo(&tsd);
```

Comment:
`self.timeseries_descriptor()` creates a copy, so it should only be called once in this method

Reply:
amended